app/helpers/pairings_helper.rb

```diff
@@ -119,7 +119,7 @@ def player_is_in_pairing(player, pairing)
   end
 
   def readable_score(pairing)
-    return '-' if pairing.score1.zero? && pairing.score2.zero?
+    return '-' if (pairing.score1.nil? && pairing.score2.nil?) || (pairing.score1.zero? && pairing.score2.zero?)
```

Comment:
I don't think this is quite enough. It might be possible for one of them to be nil but not the other one, in which case the zero check will still throw an exception. I'm going to make another PR.